src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -271,6 +271,26 @@ export default {
 		justify-content: stretch;
 		margin: 10px 8px 0 8px;
 		border-bottom: 1px solid var(--color-border);
+
+		// Override checkbox-radio-switch styles so that it looks like tabs
+		& :deep(.checkbox-radio-switch--button-variant) {
+			border: unset !important;
+			border-radius: 0 !important;
+			.checkbox-content {
+				padding: var(--default-grid-baseline);
+				border-radius: var(--default-grid-baseline) var(--default-grid-baseline) 0 0 !important;
+				margin: 0 !important;
+				border-bottom: var(--default-grid-baseline) solid transparent !important;
+				.checkbox-content__icon--checked > * {
+					color: var(--color-main-text) !important;
+				}
+			}
+			&.checkbox-radio-switch--checked .checkbox-radio-switch__content{
+				background: transparent !important;
+				color: var(--color-main-text) !important;
```

Comment:
The correct text color in that case would have been `--color-primary-element-text`

Reply:
would you mind creating a PR for this? :pray:

Reply:
done in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4834